diffuzzer/src/supervisor.rs

```diff
@@ -151,6 +162,18 @@ impl Supervisor for QemuSupervisor {
     }
 }
 
+impl Drop for QemuSupervisor {
+    fn drop(&mut self) {
+        let id = self.id.load(Relaxed);
+        if id != 0 {
+            Command::new("kill")
+                .arg(format!("{:?}", self.id))
+                .spawn()
```

Comment:
```suggestion
                .output()
```
и прочекать результат